Editorial changes all through the specification (#42)
peppelinux/draft-demarco-oauth-status-attestations: draft-demarco-oauth-status-attestations.md

Status Assertions ensure the integrity and trustworthiness of digital credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT) format, certifying their validity and non-revocation status. They function similarly to OCSP Stapling, allowing wallet instances to present time-stamped assertions from the Credential Issuer.
The approach defined in this specification allows the verification of credentials against any revocation, without direct queries to the issuer, enhancing privacy, reducing latency, and enabling offline verification. Essential for offline scenarios, Status Assertions validate digital credentials' validity, balancing scalability, security, and privacy without internet connectivity.

> Figure 1 illustrates the process by which a Wallet Instance requests a Status Assertion from the Credential Issuer and subsequently receives it.

Reviewer: ```suggestion
The figure below illustrates the process by which a Wallet Instance requests a Status Assertion from the Credential Issuer and subsequently receives it.
```

we already have the text under the figure defining its number.

Author: I usually prefer to enumerate the figures and then to specifically point to them in the text. But i guess since the image is right bellow this works as well.

~~~
Figure 1: Status Assertion Issuance Flow

> the figure below illustrates the process by which a Wallet Instance

Reviewer 2: ```suggestion
The figure below illustrates the process by which a Wallet Instance
```

check the status of a Digital Credential over time due to some privacy constraints,
in compliance with national privacy regulations.

> For instance, consider a scenario where a Verifier's repeated access to a Status List to check the revocation status of a Digital Credential could be deemed as excessive monitoring of the End-User's activities. This could potentially infringe upon the End-User's right to privacy, as outlined in Article 8 of the European Convention on Human Rights and in the the European Union's General Data Protection Regulation (GDPR), by creating a detailed profile of the End-User's interactions and credential usage without explicit consent for such continuous surveillance.

Reviewer: I'm wondering if it would be the good time to include a link to "Article 8 of the European Convention on Human Rights"

check the status of a Digital Credential over time due to some privacy constraints,
in compliance with national privacy regulations.

For instance, consider a scenario where a Verifier's repeated access to a
Status List to check the revocation status of a Digital Credential could
be deemed as excessive monitoring of the End-User's activities.
This could potentially infringe upon the End-User's right to privacy,
as outlined in [Article 8 of the European Convention on Human Rights]

> (https://www.echr.coe.int/documents/convention_eng.pdf) and

Reviewer 3: we should move these to informative references

Reviewer 4: It may be resolved by commit https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/42/commits/82be3b6d5af95e2146bcb24c5aca642a56e4dfb7

Reviewer: @SaraConsoliACN Ories is saying that instead using a markdown href it would be better to use the informative reference section if I get it right

@marinaado WDYT?

Reviewer: moved here: https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/44

the linked Status Assertion is available and presented to the
Verifier, and not expired.

> 6. **Real-time validation**: OAuth Status Lists provide the possibility to do real-time validation of the Digital Credential status. To support the real-time status validation use cases, a Wallet can implement strategy to request a new Status Assertion before sending it to the Verifier.

Author: I was thinking again about this again and perhaps we should add that this is possible in online scenarios only.

the linked Status Assertion is available and presented to the
Verifier, and not expired.

> 6. **Real-time validation**: OAuth Status Lists provide the possibility
> to do real-time validation of the Digital Credential status. To support

Reviewer 2: ```suggestion
6. **Real-time validation**: To support
```

# Proof of Possession of a Credential

The concept of Proof of Possession (PoP) of a Credential within the
framework of the Status Assertion specification encompasses a broader
perspective than merely possessing the digital bytes of the Credential.
It involves demonstrating rightful control or ownership over the
Credential, which can manifest in various forms depending on the
technology employed and the nature of the digital Credential itself.
For instance, a Credential could be presented visually (de-visu)
with a personal portrait serving as a binding element.

While this specification does not prescribe any additional methods
for the proof of possession of the Credential, it aims to offer
guidance for concrete implementations utilizing common proof of
possession mechanisms. This includes, but is not limited to:

> 1. Having the digital representation of the credential (the bytes).

Reviewer 2: This does not provide a proof of possession. It is only a necessary condition but not sufficient

+-------------------+                         +--------------------+
~~~

The Wallet Instance sends the Status Assertion request to the
Credential Issuer, where:
- The request MUST contain the base64url hash value of the Digital Credential,
for which the Status Assertion is requested, and enveloped in a signed
object as proof of possession.
- The proof of possession MUST be signed with the private key corresponding

> to the public key attested by the Credential Issuer and contained within

Reviewer 3: ```suggestion
to the confirmation claim assigned by the issuer and contained within
```

Reviewer: this text generalize the scope of the cnf claim within the jwt/cwt

this makes sense to me
